Doc/library/time.rst

```diff
@@ -815,6 +815,17 @@ These constants are used as parameters for :func:`clock_getres` and
    .. versionadded:: 3.7
 
 
+.. data:: CLOCK_UPTIME_RAW
+
+   Clock that increments monotonically, tracking the time since an arbitrary 
+   point, unaffected by frequency or time adjustments and not increment while 
+   the system is asleep.
```

Comment:
LGTM. FYI http://www.manpagez.com/man/3/clock_gettime/ has a different definition:

     CLOCK_UPTIME_RAW   clock that increments monotonically, in the same man-
                        ner as CLOCK_MONOTONIC_RAW, but that does not incre-
                        ment while the system is asleep.  The returned value
                        is identical to the result of mach_absolute_time()
                        after the appropriate mach_timebase conversion is
                        applied.

Reply:
hmmm... is the current short version better or we use the description in the man page?

Reply:
The short version is fine. The longer version doesn't add anything useful and is too low-level.